javascript/javascript1/week2/full-name.js

```diff
@@ -0,0 +1,8 @@
+function getFullname(firstname,surname){
+    return firstname+" "+surname
```

Comment:
This is not wrong but I would still recommend using string interpolation moving forward in the following homework to avoid string concatenation

Example:
```js
    return `${firstname} ${surname}`
```